src/event.js

```diff
@@ -573,7 +574,7 @@ jQuery.Event.prototype = {
 
 		this.isPropagationStopped = returnTrue;
 
-		if ( e ) {
+		if ( e && this.executeOriginalPropagation ) {
```

Comment:
In what sense is it different, though? The "Firefox <= 46" support comment could just move up here.